src/commands/run.rs

```diff
@@ -340,3 +353,53 @@ fn maybe_add_named_capability_to_store(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod unittest {
+    use crate::commands::run::{handle_run, RunArgs};
+    use rand::Rng;
+    use slight_runtime::IORedirects;
+    use std::fs::File;
+    use std::path::{Path, PathBuf};
+    use tempfile::tempdir;
+    use tokio::fs;
+
+    #[tokio::test]
+    async fn test_handle_run_with_io() -> anyhow::Result<()> {
+        let module = "./src/commands/test/io-test.wasm";
```

Comment:
I would recommend putting test artifacts to some places like `CARGO_TARGET_DIR` https://doc.rust-lang.org/cargo/reference/environment-variables.html

Reply:
No need to change it to this PR though

Reply:
Let's discuss how we'd like to build unit test wasm apps. I think we need to include them in the build.rs or something to ensure they are up-to-date when running the unit tests.

/cc @danbugs